2023/GreaterBayArea/data.json

```diff
@@ -0,0 +1,29 @@
+{
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
+      "geffzhang",
+      "kinfey"
+    ]
+  },
+  "CommunityInfo": {
+    "Name": "Global Azure GBA",
+    "Country": "China",
+    "Url": "http://hdxu.cn/qvgNP",
+    "SessionizeApiId": "",
```

Comment:
You do not intend to use Sessionize to show off your community sessions on the Global Azure web site? We support languages, in case you want to present your information in Chinese.

Reply:
yes